docs/iidm/model/phaseTapChanger.md

```diff
@@ -33,6 +33,9 @@ Each step of a phase tap changer has the following attributes:
 # Model
 A phase tap changer is regulating if **Regulating** is set to ```true```.
 
+The **Target Deadband** defines the margin withing which the target voltage is considered as respected for discrete regulation i.e. the target voltage is considered as respected
+if the read voltage equals `targetV±(targetDeadband/2)`.
```

Comment:
Are you also waiting for a positive value of the target deadband?

Reply:
Yes, I will specify it.